### PR TITLE
adding link to access page and getting started commands

### PR DESCRIPTION
**Pull request by danielwhatmuff**

### CODE REVIEW

File: app/views/integrations/_access.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      },
+      aria: {
+        controls: panel_id
+      }
+    ) do
+  %>
+    <%= icon 'key' %>
+    Setup access
+  <% end %>
+</div>
```
Comment (jits) [inline app/views/integrations/_access.html.erb]: Given this is just a link, and used in one place, you don't need this partial.

Also, you don't need the panel_id stuff, and can greatly simplify the way the link tag is rendered.

Comment (danielwhatmuff) [reply app/views/integrations/_access.html.erb]: Sure I can add directly.

File: app/views/resources/_list-item.html.erb [hunk 2]
```diff
@@ -48,10 +48,25 @@
       <%- end -%>
     </div>
 
+    <div class="indented mt-3">
```
Comment (danielwhatmuff) [inline app/views/resources/_list-item.html.erb]: Moved to here @jits

Comment (jits) [reply app/views/resources/_list-item.html.erb]: 👍

**Comment by jits** [comment]
I think this is ready to go?

**Pull request merged**